oneflow/core/job/env_global_objects_scope.cpp

```diff
@@ -83,8 +92,9 @@ Maybe<void> EnvGlobalObjectsScope::Init(const EnvProto& env_proto) {
   Global<EnvDesc>::New(env_proto);
   Global<CtrlServer>::New();
   Global<ProcessCtx>::New();
-  JUST(HostListCtrlBootstrap(*Global<EnvDesc>::Get())
-           .InitProcessCtx(Global<CtrlServer>::Get()->port(), Global<ProcessCtx>::Get()));
+  // avoid dead lock with multi nodes;
```

Comment:
Avoid dead lock by using CHECK_JUST instead of JUST. because it maybe be blocked in ~CtrlBootstrap